react/src/components/dropdown/SprkDropdown.js

```diff
@@ -147,37 +175,37 @@ class SprkDropdown extends Component {
                   sprk-c-Icon--stroke-current-color
                   sprk-u-mls
                 "
```

Comment:
We should allow the `iconAddiontionalClasses` to be added to both the default and informational variants.